google/cloud/security/inventory/inventory_loader.py

```diff
@@ -172,8 +171,7 @@ def _run_pipelines(pipelines):
             pipeline.run()
             pipeline.status = 'SUCCESS'
             LOGGER.info('Finished running %s', pipeline.__class__.__name__)
-        except (api_errors.ApiInitializationError,
-                inventory_errors.LoadDataPipelineError) as e:
+        except Exception as e:
```

Comment:
Ah, I do see self._no_organization() being called in the try now, which will exit.  Yup, that will work as is.  Thanks!  :)